src/main.rs

```diff
@@ -87,6 +87,10 @@ impl KubernetesMetrics {
                     metric.set_fs_used_bytes(fs_used_bytes);
                 }
 
+                if let (Some(fs_available_bytes), Some(fs_used_bytes)) = (json["fs"]["availableBytes"].as_f64(), json["fs"]["usedBytes"].as_f64()) {
+                    metric.set_disk_usage(fs_used_bytes / fs_available_bytes);
```

Comment:
You'll probably have to make it more hufter proof (division by zero prevention, no negative value, and no value > 100% :))

Reply:
(Talking from experience here :P)